tsconfig.json

```
@@ -0,0 +1,25 @@
+    "target": "es2022",
+    "module": "es2022",
+    "moduleResolution": "bundler",
+    "lib": ["ES2020", "ESNext.Array", "DOM"],
+    "outDir": "dist",
+    "rewriteRelativeImportExtensions": true,
+    "isolatedModules": true,
+    "noEmitOnError": true,
+    "strict": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "exactOptionalPropertyTypes": true,
```

Biome (1.9.4)

[error] The key exactOptionalPropertyTypes was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)